src/routes/lobby/LobbyView.vue

```diff
@@ -217,7 +217,9 @@ export default {
   padding: 0 64px;
   width: 50%;
 }
-
+.rank-switch * {
+  flex-shrink: 0;
+}
```

Comment:
I think it's better to put `flex-shrink-0` as a vuetify utility class directly on the v-swtich